src/e2ee/RoomTracker.ts

```diff
@@ -61,7 +52,10 @@ export class RoomTracker {
             encEvent = await this.client.getRoomStateEvent(roomId, "m.room.encryption", "");
             encEvent.algorithm = encEvent.algorithm ?? 'UNKNOWN';
         } catch (e) {
-            return; // failure == no encryption
+            if (e instanceof MatrixError && e.errcode === "M_NOT_FOUND") {
+                encEvent = {};
+            }
+            return; // Other failures should not be cached.
```

Comment:
Should this be in an `else` branch?  Without it, it looks like the empty `encEvent` never gets used/cached.

Reply:
oh yes, duh.